src/block/tab-labels/edit.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames'
+import { InspectorTabs } from '~stackable/components'
+import {
+	BlockDiv,
+	useGeneratedCss,
+	Alignment,
+	Advanced,
+	CustomCSS,
+	Responsive,
+	CustomAttributes,
+	EffectsAnimations,
+	ConditionalDisplay,
+	Separator,
+	Transform,
+} from '~stackable/block-components'
+import {
+	withBlockAttributeContext,
+	withBlockWrapperIsHovered,
+	withQueryLoopContext,
+} from '~stackable/higher-order'
+// import { version as VERSION, i18n } from 'stackable'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+import { dispatch, select } from '@wordpress/data'
+import { attributes } from '../tabs/schema'
+import { getBlockFromExample } from '@wordpress/blocks'
+
+export const checkOpen = i => {
+	if ( i === 1 ) {
+		return true
+	}
+	return false
+}
+
+const Edit = props => {
+	const {
+		className,
+		isSelected,
+	} = props
+
+	useGeneratedCss( props.attributes )
+
+	const {
+		insertBlock, updateBlockAttributes,
+	} = dispatch( 'core/block-editor' )
+
+	const {
+		getBlockRootClientId, getNextBlockClientId,
+	} = select( 'core/block-editor' )
+
+	const tabContent = getNextBlockClientId( props.clientId )
+	const tabsBlock = getBlockRootClientId( props.clientId )
```

Comment:
This will slow down the editor because it will traverse the block tree every render. I don't think you need to know the `tabContent` and `tabsBlock` every time, and I think you'll only need to reference these when changing/clicking on tabs. If so, then the best way would be only do these during the action itself.